features/0794-did-rotate/README.md

```diff
@@ -0,0 +1,137 @@
+# Aries RFC 0794: DID Rotate 1.0
+
+- Authors: [Sam Curren](telegramsam@gmail.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2023-08-29 
+- Status Note: 
+- Start Date: 2023-08-18 
+- Tags: [feature](/tags.md#feature), [protocol](/tags.md#protocol)
+- URI: https://didcomm.org/did-rotate/1.0
+
+## Summary
+
+This protocol signals the change of DID in use between parties.
+
+This protocol is only applicable to DIDComm v1 - in DIDComm v2 use the more efficient DID Rotation header.
+
+## Motivation
+
+This mechanism allows a party in a relationship to change the DID in use for the relationship. This may be used to switch DID methods,
+but also to switch to a new DID within the same DID method. Inspired by (but different from) the DID rotation feature of the DIDComm Messaging (DIDComm v2) spec.
+
+## Implications for Software Implementations
+
+This protocol enables the identifiers used in a relationship to change. Implementations will need to consider how data related to the relationship is managed. If the relationship DIDs are used as identifiers, those identifiers may need to be updated during the rotation to maintain data integrity.
+
+## Tutorial
+
+### Name and Version
+
+DID Rotate 1.0
+
+URI: https://didcomm.org/did-rotate/1.0/<messageType>
+
+### Roles
+
+**rotating_party**: this party is rotating the DID in use for this relationship. They send the `rotate` message.
+
+**observing_party**: this party is notified of the DID rotation
+
+### Messages
+
+#### Rotate 
+
+Message Type URI: https://didcomm.org/did-rotate/1.0/rotate
+
+`to_did`: The new DID to be used to identify the **rotating_party**
+
+```json
+{
+    "@id": "123456780",
+    "@type": "https://didcomm.org/did-rotate/1.0/rotate",
+    "to_did": "did:example:newdid",
+
+}
+```
+
+The **rotating_party** is expected to receive messages on both the existing and new DIDs and their associated keys for a reasonable period that MUST extend at least until the following `ack` message has been received.
+
+This message MUST be sent using authcrypt or as a signed message in order to establish the provenance of the new DID. Proper provenance prevents injection attacks that seek to take over a relationship. Any rotate message received without being authcrypted or signed MUST be discarded and not processed.
```

Comment:
I would add something like so that readers unfamiliar with the DIDComm message handling will not go down a rabbit hole of trying to figure out if a message is AuthCrypt or not: “In Aries implementations, messages sent in the context of an existing relationship are by default sent using authcrypt.”